app/build.gradle

```diff
@@ -64,6 +85,18 @@ check {
 
 jacocoTestReport {
     dependsOn test // tests are required to run before generating the report
+
+    afterEvaluate {
+        classDirectories.setFrom(files(classDirectories.files.collect {
+            fileTree(dir: it, exclude:
+                    ['pl/futurecollars/invoicing/controller/InvoiceController.class',
+                    'pl/futurecollars/invoicing/InvoiceApplication.class',
+                    'pl/futurecollars/invoicing/db/FileBasedDatabase.class',
+                    'pl/futurecollars/invoicing/db/IdService.class',
+                    'pl/futurecollars/invoicing/utils/JsonService.class',
+                    'pl/futurecollars/invoicing/utils/FileService.class'])
+        }))
+    }
```

Comment:
a spróbuj dodać dokładnie to samo w sekcji jacocoTestCoverageVerification i sprawdź czy wtedy przejdzie  bez generated :)